tsconfig.json

```diff
@@ -4,6 +4,7 @@
     "target": "es6",
     "sourceMap": true,
     "outDir": "dist",
+    "declaration": true,
```

Comment:
you also should add `"types": "./dist/fs.d.ts"` in `package.json`

Reply:
Done :)